source/community/events/archives/workshop/workshop-march-2012.html.md

```diff
@@ -31,7 +31,7 @@ wiki_last_updated: 2012-03-22
 
 ### Slides
 
-[Media:Workshop-1203-vdsm-mom.odp‎](Media:Workshop-1203-vdsm-mom.odp‎)
+[Media:Workshop-1203-vdsm-mom.odp](Media:Workshop-1203-vdsm-mom.odp)
```

Comment:
This is some weird MediaWiki-generated fake link. It won't work and needs further rewriting to do so.